zeppelin-server/src/main/java/org/apache/zeppelin/realm/LdapRealm.java

```diff
@@ -217,7 +254,7 @@ protected AuthenticationInfo doGetAuthenticationInfo(AuthenticationToken token)
    */
   @Override
   protected AuthenticationInfo queryForAuthenticationInfo(AuthenticationToken token,
-          LdapContextFactory ldapContextFactory) throws NamingException {
```

Comment:
There were a lot of empty spaces, have tried to clean up all of those with this PR.